moto/mediastoredata/models.py

```diff
@@ -0,0 +1,113 @@
+from __future__ import unicode_literals
+
+import datetime
+from collections import OrderedDict
+
+from boto3 import Session
+
+from moto.core import BaseBackend, BaseModel
+from .exceptions import ClientError
+
+
+class CreatedObject(BaseModel):
+    def __init__(self, *args, **kwargs):
```

Comment:
If there are only a few, passing in named arguments here will be easier to read then having to parse a dict